xbmc/guilib/GUIFontManager.cpp

```diff
@@ -142,37 +153,41 @@ CGUIFont* GUIFontManager::LoadTTF(const std::string& strFontName,
   }
 
   // check if we already have this font file loaded (font object could differ only by color or style)
-  std::string TTFfontName =
+  const std::string fontIdent =
       StringUtils::Format("{}_{:f}_{:f}{}", strFilename, newSize, aspect, border ? "_border" : "");
 
-  CGUIFontTTF* pFontFile = GetFontFile(TTFfontName);
+  std::shared_ptr<CGUIFontTTF> pFontFile = GetFontFile(fontIdent);
   if (!pFontFile)
   {
-    pFontFile = CGUIFontTTF::CreateGUIFontTTF(TTFfontName);
+    pFontFile = CGUIFontTTF::CreateGUIFontTTF(fontIdent);
     bool bFontLoaded = pFontFile->Load(strPath, newSize, aspect, 1.0f, border);
 
     if (!bFontLoaded)
     {
-      delete pFontFile;
-
       // font could not be loaded - try Arial.ttf, which we distribute
       if (strFilename != "arial.ttf")
       {
-        CLog::Log(LOGERROR, "Couldn't load font name: {}({}), trying to substitute arial.ttf",
-                  strFontName, strFilename);
-        return LoadTTF(strFontName, "arial.ttf", textColor, shadowColor, iSize, iStyle, border, lineSpacing, originalAspect);
+        CLog::Log(
+            LOGERROR,
```

Comment:
This formatting seems odd.

Reply:
There was done by the clang cleanup and no direct idea to make shorter.

Maybe as:
```cpp
        CLog::Log(LOGERROR, "GUIFontManager::{}: Couldn't load font name: {}({}), trying arial.ttf",
                  __func__, strFontName, strFilename);
```
with the reduced length bring clang it as this.

Reply:
Have added the reduced length. If not OK then I remove again.